File: Final Project/HotelManagement_API/Model/Room.cs
namespace HotelManagement.Model
{
    public class Room
    {
        public int RoomId { get; set; }
        public string? RoomNumber { get; set; }
        public string? RoomType { get; set; }
        public decimal Price { get; set; }
        public string? Status { get; set; }
        public string? Description { get; set; }
        public int Capacity { get; set; }
        public DateTime DateCreated { get; set; }
        public bool AC { get; set; }
        public string? RoomImage { get; set; }

        // Navigation properties
        public ICollection<Reservation>? Reservations { get; set; }
    }

}

File: Final Project/HotelManagement_Angular/src/app/Room.ts
export interface Room{
    roomId: number;
    roomNumber: string;
    roomType: string;
    price: number;
    status: string;
    description: string;
    capacity: number;
    dateCreated: Date;
    ac: boolean;
    roomImage: string;
    reservations: any[];
}